tests/test_core.py

```diff
@@ -7335,15 +7341,20 @@ def test_async(self):
   }
 }
 '''
-      open('post.js', 'w').write(r'''
-ccall('stringf', 'string', ['string'], ['first\n'], { async: true })
-  .then(function(val) {
-    Module.print(val);
-    ccall('floatf', 'number', null, null, { async: true }).then(Module.print);
-  });
+      open('pre.js', 'w').write(r'''
+Module['onRuntimeInitialized'] = function() {
+  ccall('stringf', 'string', ['string'], ['first\n'], { async: true })
+    .then(function(val) {
+      console.log(val);
+      ccall('floatf', 'number', null, null, { async: true }).then(console.log);
+    });
+};
 ''')
       self.do_run(src, 'first\nsecond\n6.4');
 
+  def test_async_emterpretify(self, emterpretify=False):
```

Comment:
Looks like the second arg isn't need here?

Reply:
Thanks, fixing.